src/program-common/person_hsv2.h

```diff
@@ -28,6 +28,11 @@ class Person_HSV2
 
 	double getHazardAParameter() const												{ return m_hazardAParam; }
 
+	double getHazardB2Parameter() const
+								{ return
+m_hazardB2Param; }
+
+
```

Comment:
Please put all of this on a single line, and remove the empty line between getHazardAParameter and getHazardB2Parameter